cura/PrintInformation.py

```diff
@@ -161,8 +160,13 @@ def _calculateInformation(self):
                     else:
                         cost = 0
 
+            radius = Application.getInstance().getGlobalContainerStack().getProperty("material_diameter", "value") / 2
```

Comment:
Is it possible to get the radius just once outside the `for`-loop like the old code? Or it will cause problems?